yarn-project/circuit-types/src/interfaces/prover-broker.ts

```diff
@@ -39,9 +40,14 @@ export const proverBrokerConfigMappings: ConfigMappingsType<ProverBrokerConfig>
     description: 'If starting a prover broker locally, the max number of retries per proving job',
     ...numberConfigHelper(3),
   },
-  proverBrokerDataDirectory: {
-    env: 'PROVER_BROKER_DATA_DIRECTORY',
-    description: 'If starting a prover broker locally, the directory to store broker data',
+  dataDirectory: {
```

Comment:
Can `...dataConfigMappings` be done instead?

Reply:
Not without adding a dependency on `@aztec/kv-store` to `@aztec/circuit-types`

Reply:
Hmm, it feels wrong then that `circuit-types` contains configuration types for apps. It seems much too low level for that.